src/api/Config/types.ts

```diff
@@ -208,29 +208,23 @@ export interface INeo4jSearchConfig extends IVendorConfig {
   indexEdges?: boolean;
 }
 
-export type InternalIndexConfig = IElasticSearchConfig | IElasticSearch2Config;
+export type InternalIndexConfig = IElasticSearchConfig;
 
-export interface ICommonElasticSearchConfig extends IVendorConfig {
+export interface IElasticSearchConfig extends IVendorConfig {
   host: string;
   port: number;
   https?: boolean;
   user?: string;
   password?: string;
-  forceReindex?: boolean;
-  skipEdgeIndexation?: boolean;
+  mapping?: string;
   dynamicMapping?: boolean;
+  forceStringMapping?: string[];
   analyzer?: string;
   incrementalIndexation?: boolean;
   timestampPropertyName?: string;
-}
-
-export interface IElasticSearchConfig extends ICommonElasticSearchConfig {
-  indexName?: string; // Deprecated option to be removed, kept here for backward compatibility
-}
-
-export interface IElasticSearch2Config extends ICommonElasticSearchConfig {
-  forceStringMapping?: string[];
-  caCert?: string;
```

Comment:
what is exactly the `caCert` config and why its removed ?

Reply:
it's an old version of advanced.extraCertificateAuthorities that was working only with Elasticsearch.
Now it works for every external service.
No comment on the main PR? https://github.com/Linkurious/linkurious-server/pull/2668

Reply:
@francescoinfante I haven't reviewed it yet but probably i will start reviewing it today :)